packages/theming/android-theme/src/androidTypography.ts

```diff
@@ -0,0 +1,51 @@
+import { FontSize, FontSizes, FontWeightValue, Typography, Variants } from '@fluentui-react-native/theme-types';
+
+export function androidTypography(): Typography {
+  const androidDict = {
+    sizes: {
+      caption: 12 as FontSize,
+      secondary: 11 as FontSize,
+      body: 13 as FontSize,
+      subheader: 16 as FontSize,
+      header: 20 as FontSize,
+      hero: 22 as FontSize,
+      heroLarge: 26 as FontSize,
+    } as FontSizes,
+    weights: {
+      ultralight: '100' as FontWeightValue,
+      thin: '200' as FontWeightValue,
+      light: '300' as FontWeightValue,
+      regular: '400' as FontWeightValue,
+      medium: '500' as FontWeightValue,
+      semiBold: '600' as FontWeightValue,
+      bold: '700' as FontWeightValue,
+      heavy: '800' as FontWeightValue,
+      black: '900' as FontWeightValue,
+    },
+    families: {
+      primary: 'SF Pro',
+      secondary: 'System',
+      cursive: 'System',
+      monospace: 'SF Mono',
+      sansSerif: 'SF Pro',
+      serif: 'System',
+    },
+    variants: {
+      captionStandard: { face: 'primary', size: 'caption', weight: '500' },
+      secondaryStandard: { face: 'primary', size: 'secondary', weight: '400' },
+      secondarySemibold: { face: 'primary', size: 'secondary', weight: '600' },
+      bodyStandard: { face: 'primary', size: 'body', weight: '400' },
+      bodySemibold: { face: 'primary', size: 'body', weight: '600' },
+      subheaderStandard: { face: 'primary', size: 'subheader', weight: '400' },
+      subheaderSemibold: { face: 'primary', size: 'subheader', weight: '600' },
+      headerStandard: { face: 'primary', size: 'header', weight: '700' },
+      headerSemibold: { face: 'primary', size: 'header', weight: '800' },
+      heroStandard: { face: 'primary', size: 'hero', weight: '400' },
+      heroSemibold: { face: 'primary', size: 'hero', weight: '700' },
+      heroLargeStandard: { face: 'primary', size: 'heroLarge', weight: '400' },
+      heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: '700' },
```

Comment:
These look apple specific. Specifically "SF Pro", and all the font weights. Should probably remove

Reply:
Changed Typography as per [Figma Fluent Android](https://www.figma.com/file/MkkE13z6zALstwLlbczIMXrA/Fluent-Android?node-id=7455%3A94).